src/network/networkd-dhcp6.c

```diff
@@ -646,8 +646,14 @@ static int dhcp6_configure(Link *link) {
 
         r = sd_dhcp6_client_set_prefix_delegation(client, link->network->dhcp6_use_pd_prefix);
         if (r < 0)
-                return log_link_debug_errno(link, r, "DHCPv6 CLIENT: Failed to %s prefix delegation: %m",
-                                            enable_disable(link->network->dhcp6_use_pd_prefix));
+                return log_link_debug_errno(link, r, "DHCPv6 CLIENT: Failed %s to request prefix to be delegated: %m",
```

Comment:
`Failed not to request prefix to be delegated` doesn't sound like what you want, maybe `Failed to request that prefix does %s get delegated`?

Reply:
Not sure about the difference... When the flag is off, then the dhcp6 client does not request IA_PD.
How about the following: "Failed to enable/disable to request prefix to be delegated: %m"?

Reply:
I'd say `Failed to enable/disable requesting for prefix to be delegated`

Reply:
Thanks.